about.html

```diff
@@ -0,0 +1,169 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Lato"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/51f27b15da.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Abrehot Library</title>
+  </head>
+  <body>
+    <header class="font">
+      <nav class="star">
+        <i class="fa-sharp fa-solid fa-bars fa-xl hamburger"></i>
+      </nav>
+      <nav class="first-nav">
+        <ul>
+          <li>
+            <a href="#"><i class="fa-brands fa-facebook-f"></i></a>
+          </li>
+          <li>
+            <a href="#"><i class="fa-brands fa-twitter"></i></a>
+          </li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Page</a></li>
+          <li><a href="#">Logout</a></li>
+        </ul>
+      </nav>
+      <nav class="second-nav">
+        <div>
+          <a href="index.html"><h2>Abrehot Library</h2></a>
+        </div>
+        <div>
+          <ul>
+            <li><a href="about.html">About</a></li>
+            <li><a href="#">Books</a></li>
+            <li><a href="#">Membership</a></li>
+            <li><a href="#">Events</a></li>
+            <li><a href="#">Launched</a></li>
+            <li class="last-item"><a href="#">Library</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <!-- Mobile menu -->
+    <section class="mobile-menu font">
+      <button class="x-mark">&times;</button>
+      <ul id="menu-list">
+        <li><a href="index.html">Home</a></li>
+        <li><a href="about.html">About</a></li>
+        <li><a href="#">Contact</a></li>
+      </ul>
+    </section>
+    <!-- About page Introduction -->
+    <main class="bg-gray font">
+      <section class="intro ab-intro bg-gray font">
+        <h1>"Unleash your curiosity at the library."</h1>
+        <div class="border-1 about-intro bg-gray">
+          <p>
+            The Abrehot Library, established by Prime Minister Abiy Ahmed, is a
+            modern facility designed to provide a range of resources and
+            services to the public. The library offers a wide range of books
+            that cover various topics, from politics and history to literature
+            and science, available for borrowing. Additionally, it has a unique
+            feature, which is the reading corner. This cozy and inviting space
+            is designed to encourage reading and reflection, offering
+            comfortable chairs and cushions, and natural light, creating a warm
+            and welcoming atmosphere.
+          </p>
```

Comment:
- As Requested by your Previous reviewer, Please Follow [this Review](https://github.com/Yonasle/Abrehot_Capstone/pull/1#:~:text=Following%20the%20design%20template%20we%20should%20have%20a%20large%20logo%20below%20the%20paragraph%2C%20however%2C%20if%20you%20don%27t%20have%20a%20logo%20you%20can%20use%20an%20image.&text=Good%20job%20by%20adding%20a%20partner%20section%20as%20required%20by%20the%20previous%20reviewer%20but%20I%20think%20you%20can%20improve%20your%20design%20because%20it%20looks%20different%20from%20the%20original%20design.%20Please%20make%20sure%20to%20stick%20to%20the%20original%20design%20to%20give%20a%20good%20user%20experience%20and%20make%20your%20website%20great&text=Following%20the%20design%20template%20we%20should%20have%20a%20large%20logo%20below%20the%20paragraph%2C%20however%2C%20if%20you%20don%27t%20have%20a%20logo%20you%20can%20use%20an%20image.) And make Sure to add a logo or an image under this section. 
**Original Design**
![234961947-99f14c91-dd3f-44ec-906a-240a8ebba605](https://user-images.githubusercontent.com/97834160/235117155-bca1da3c-2796-4308-8a35-bbe21eaa04b9.png)

**Your Design**
![Screenshot from 2023-04-28 11-53-53](https://user-images.githubusercontent.com/97834160/235117264-3a13b157-9c71-4bce-838a-d4abdfb6b4e1.png)